Project.toml

```diff
@@ -1,18 +1,32 @@
-name = "StatisticalTraits"
-uuid = "64bff920-2084-43da-a3e6-9bb72801c0c9"
+name = "ScientificTypes"
```

Comment:
???

Surely you don't mean to make all these changes to the Project.toml (including changing the name of the StatisticalTraits package)? You just want to replace one of the deps.

Reply:
Yeah. An error on my part.